com.unity.render-pipelines.core/Runtime/Utilities/FSRUtils.cs

```diff
@@ -86,25 +86,42 @@ public static void SetEasuConstants(CommandBuffer cmd, Vector2 inputViewportSize
             cmd.SetGlobalVector(ShaderConstants._FsrEasuConstants3, constants3);
         }
 
+        /// <summary>
+        /// The maximum sharpness value in stops before the effect of RCAS is no longer visible.
+        /// This value is used to map between linear and stops.
+        /// </summary>
+        internal const float kMaxSharpnessStops = 2.5f;
+
+        /// <summary>
+        /// AMD's FidelityFX Super Resolution integration guide recommends a value of 0.2 for the RCAS sharpness parameter when specified in stops
+        /// </summary>
+        public const float kDefaultSharpnessStops = 0.2f;
```

Comment:
this could be internal as well?

Reply:
I don't think I can make these internal since they're located in core and used by URP? I get accessibility compiler errors when I try.

Reply:
ah ok, I just looked at the diffs here and it looked like it wasn't used by other packages. Then public is fine, thanks for checking :)